qtile/management/commands/process_screenshots.py

```diff
@@ -14,16 +14,12 @@
 class Command(BaseCommand):
     def handle(self, *args, **kwargs):
         for screenshot in models.Screenshot.load():
-            print screenshot.name
-            img = Image.open(staticfiles_storage.path(screenshot.path))
-
-            # Ensure we're in RGB
-            if img.mode not in ('L', 'RGB'):
-                img = img.convert('RGB')
+            print(screenshot.name)
```

Comment:
Debugging?

Reply:
This script runs as part of the build process on Heroku, and prints the filename for potential debugging purposes so that, e.g., if PIL is unable to read an image we have an easy way to check the logs and see which screenshot it blew up on.

(The `img.mode` check was moved into the `Screenshot` class.)